osu.Game/Overlays/VolumeOverlay.cs

```diff
@@ -43,11 +43,16 @@ private void load(AudioManager audio, OsuColour colours)
 
             AddRange(new Drawable[]
             {
-                new Box
+                new SafeAreaContainer
                 {
-                    RelativeSizeAxes = Axes.Y,
-                    Width = 300,
-                    Colour = ColourInfo.GradientHorizontal(Color4.Black.Opacity(0.75f), Color4.Black.Opacity(0))
+                    RelativeSizeAxes = Axes.Both,
+                    SafeAreaOverrideEdges = Edges.All,
```

Comment:
Why is `Edges.Right` included here? From what I can tell it should work fine without.

Reply:
The background Box needs to snap to all edges of the screen.